lib/src/realm_class.dart

```diff
@@ -60,6 +61,8 @@ export 'package:realm_common/realm_common.dart'
         ObjectId,
         Uuid;
 
+export 'package:cancellation_token/cancellation_token.dart' show CancellationToken, CancelledException;
```

Comment:
Why this export?

Reply:
So that people don't need to import an additional package to use our API. I believe it's similar to what we do for the datatypes.

Reply:
You are thinking about `Uuid` and `ObjectId` from `realm_common`?

Reply:
Yes

Reply:
I'm not a fan of those exports either 😃 .. 

Anyway, I have made my view-point known, and we will let the community judge 😄